README.md

```diff
@@ -131,12 +131,13 @@ headless = Headless.new(:video => { :frame_rate => 12, :codec => 'libx264' })
 Available options:
 
 * :codec - codec to be used by ffmpeg
-* :frame_rate    - frame rate of video capture
-* :provider      - ffmpeg provider - either :libav (default) or :ffmpeg
-* :pid_file_path - path to ffmpeg pid file, default: "/tmp/.headless_ffmpeg_#{@display}.pid"
-* :tmp_file_path - path to tmp video file,  default: "/tmp/.headless_ffmpeg_#{@display}.mov"
-* :log_file_path - ffmpeg log file,         default: "/dev/null"
-* :extra         - array of extra ffmpeg options, default: [] 
+* :frame_rate      - frame rate of video capture
+* :provider        - ffmpeg provider - either :libav (default) or :ffmpeg
+* :provider_binary - Explicit path to avconv or ffmpeg.  Only required when the binary cannot be discovered on the system $PATH.
```

Comment:
I'm fine w/ any of those names.  I stuck w/ ```provider_binary``` simply because it allowed the code to remain backwards-compatible w/ the previous version.  (That is: the version could be incremented to 2.1.0 instead of 3.0.0).  But, if you'd prefer to choose a different name, then just let me know which one you'd like, and I can update to code accordingly.